modules/ossm-running-v2-v3-multitenant-deployment-model.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-v2-v3-side-by-side-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-running-v2-v3-multitenant-deployment-model_{context}"]
+= Running {SMProductName} 2 and 3 using multi-tenant deployment model
+//TP1 content influx. Title, etc may change.
+//No IA as of 10/23/2024 so this content is likely to move.
+
+If you are moving from {SMProductName} 2.6 in a multi-tenant deployment model, you can run {SMProduct} 2.6 side-by-side with {SMProduct} 3.0, in one cluster, without them interfering with each other.
```

Comment:
```suggestion
If you are moving from {SMProductName} 2.6 in a multi-tenant deployment model (the default model), you can run {SMProduct} 2.6 side-by-side with {SMProduct} 3.0, in one cluster, without them interfering with each other.
```

Reply:
We should call out that this is the default model, because I can see a lot of users thinking... "I'm not doing anything multi-tenant" and not realizing that they are in fact using the multi-tenant model if they have not explicitly configured cluster-wide... they just never thought about it. I would guess this is 80% of users.